### ISSUE TITLE
Strange Display Problems

### ISSUE THREAD

**Issue by KeygenOld** [thejoshsmith/craft-prism-syntax-highlighting#21]
I'm using Firefox (latest) on macOS, latest Craft CMS, and running locally on MAMP Pro. Having strange display issues.
- Theme is brown regardless of settings
- Line numbers do not show up
- Strange blue outlines around all the code (looks like ugly blue outline you get on input fields that have `:focus`)

Any ideas as to what would cause this?

<img width="515" alt="prism_display" src="https://user-images.githubusercontent.com/92394639/139161680-6d4fc34e-9215-496b-bb63-dc6afb45c64a.png">

<img width="552" alt="prism_settings" src="https://user-images.githubusercontent.com/92394639/139161691-f5594ede-807c-42e8-9a60-1e3454398bcf.png">

**Comment by adrienne** [comment]
Having the same issue here.

**Comment by engram-design** [comment]
This is fixed in [2.0.0](https://github.com/verbb/prism/releases/tag/2.0.0). Do note however that the plugin has changed to `verbb/prism`.

Run the following:

```shell
php craft plugin/uninstall craft-prism-syntax-highlighting
composer require verbb/prism && composer remove thejoshsmith/craft-prism-syntax-highlighting
php craft plugin/install prism
```

The new plugin will migrate your fields across.
